lib/middleware/karma.js

```diff
@@ -32,9 +33,17 @@ var CROSSORIGIN_ATTRIBUTE = 'crossorigin="anonymous"'
 var LINK_TAG_CSS = '<link type="text/css" href="%s" rel="stylesheet">'
 var LINK_TAG_HTML = '<link href="%s" rel="import">'
 var SCRIPT_TYPE = {
-  '.js': 'text/javascript',
-  '.dart': 'application/dart'
+  'js': 'text/javascript',
+  'dart': 'application/dart',
+  'esModule': 'module'
 }
+var FILE_TYPES = [
+  'css',
+  'html',
+  'js',
+  'dart',
+  'esModule'
```

Comment:
This is total bikeshedding, but this could be more simply called `module`.  It removes the only camel-cased option, and makes the API more consistent with the browser: `type: 'module'`.

Reply:
That's fine I was just copying from the previous PR but I think `'module'` is better. Fixed now